src/styles/blocks/container.scss

```diff
@@ -0,0 +1,27 @@
+.container {
+  font-family: Roboto, Arial, sans-serif;
+
+  box-sizing: border-box;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-rows: repeat(2, 1fr);
+  justify-content: center;
+  align-items: center;
+  gap: 46px 48px;
+
+  @include set-column-count(1);
+
+  @include screen-s {
+    @include set-column-count(2);
+  }
+
+  @include screen-m {
+    @include set-column-count(3);
+  }
+
+  @include screen-l {
```

Comment:
Ensure that the mixin 'screen-l' is defined in your project. If it's not defined, this will cause an error.